alfalfa_worker/worker_fmu/worker.py

```diff
@@ -158,165 +70,48 @@ def step_sim_type(self, site_id, step_sim_type, step_sim_value, start_datetime,
         :param step_sim_value:
         :param start_datetime:
         :param end_datetime:
-        :param model_type: string, type of model. Choices are 'osm' or 'fmu'
         :return:
         """
-        self.logger.info("Calling model '{}' step_sim_type '{}'".format(model_type, step_sim_type))
+        self.logger.info("Calling model '{}' step_sim_type 'fmu'".format(step_sim_type))
```

Comment:
should make `fmu` a member variable of the class allowing for copy/paste as needed.

Reply:
Oops, I also just realized I replaced sim type instead of model type in the print.